save-api/src/main/README.md

```diff
@@ -0,0 +1,30 @@
+# Save Cloud API Library
```

Comment:
Please move this file into `save-api`; documentation isn't usually placed in `src`.

Reply:
Thanks, misplaced, fixed